library/test/src/cli.rs

```diff
@@ -281,6 +281,10 @@ fn parse_opts_impl(matches: getopts::Matches) -> OptRes {
 
     let options = Options::new().display_output(matches.opt_present("show-output"));
 
+    if logfile.is_some() && !format.is_programmatic() {
+        let _ = write!(io::stdout(), "warning: `--logfile` is deprecated");
```

Comment:
Why is this warning printed to stdout instead of stderr?

Reply:
Fixed